schemas/dependencies/jenkins-instance-1.yml

```diff
@@ -50,6 +50,39 @@ properties:
       required:
       - name
       - keep_hours
+  workerFleets:
+    type: array
+    items:
+      type: object
+      additionalProperties: false
+      properties:
+        account:
+          "$ref": "/aws/tenant_accounts-1.yml#/properties/account"
+        identifier:
+          "$ref": "/common-1.json#/definitions/longIdentifier"
+        namespace:
+          "$ref": "/common-1.json#/definitions/crossref"
+          "$schemaRef": "/openshift/namespace-1.yml"
+        credentialsId:
+          type: string
+        fsRoot:
+          type: string
+        labelString:
+          type: string
```

Comment:
I guess it's important to state here that this has to match the tags given in the `asg` configuration